tests/frontend_selenium/tests/test_farm.py

```diff
@@ -218,61 +149,29 @@ def test_add_invalid_farmpayout_address(browser):
     for case in cases:
         assert farm_page.add_farmpayout_address(case).is_enabled()==False
         assert farm_page.wait_for('Edit')
-
-
-def test_edit_farmpayout_address(browser):
-    """
-    Test Case: TC916 - Edit farm payout address
-    Steps:
-        - From the Twin Details Page Click on Farms
-        - Make sure you have at least one farm.
-        - Click on the arrow behind ant farm id
-        - Click on the edit v2 address button
-        - Put the address you got
-        - Click Submit
-    Result: You should see the bottom right alert with the message "address added"
-    """
-    farm_page, polka_page, farm_name, password = before_test_setup(browser)
-    farm_page.create_farm(farm_name)
-    polka_page.authenticate_with_pass(password)
-    assert farm_page.wait_for('Farm created!')
-    assert farm_page.wait_for(farm_name)
     case = "GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG"
-    farm_page.setup_farmpayout_address(farm_name)
     farm_page.add_farmpayout_address(case).click()
     polka_page.authenticate_with_pass(password)
+    assert farm_page.wait_for('Transaction submitted')
     assert farm_page.wait_for('Address added!')
+    assert farm_page.wait_for('Edit')
     assert farm_page.wait_for('GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG')
+    farm_page.close_detail()
+    case = "GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4"
+    farm_page.setup_farmpayout_address(farm_name)
+    farm_page.add_farmpayout_address(case).click()
+    polka_page.authenticate_with_pass(password)
+    assert farm_page.wait_for('Transaction submitted')
+    assert farm_page.wait_for('Address added!')
+    assert farm_page.wait_for('GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4')
 
 
-def test_valid_ip(browser):
+def test_ip(browser):
```

Comment:
All ip related tests in the test_farm file are failing.

![image](https://user-images.githubusercontent.com/101194226/223993560-00d1368b-ed51-485d-9958-3ce5ae8ddb3e.png)

Full logs can be found [here](https://gist.github.com/mohamedamer453/10ea68cb1ecd6285bd73ec8d1d458e45).